src/ansys/grantami/recordlists/_models.py

```diff
@@ -906,6 +905,7 @@ class AuditLogAction(str, Enum):
     LISTPUBLISHED = models.GsaListAction.LISTPUBLISHED.value
     LISTUNPUBLISHED = models.GsaListAction.LISTUNPUBLISHED.value
     LISTREVISIONCREATED = models.GsaListAction.LISTREVISIONCREATED.value
+    LISTREVISIONDISCARDED = models.GsaListAction.LISTREVISIONDISCARDED.value
```

Comment:
I think this action was a late addition. If we received an audit log with this action included, we'd probably fail to represent it here.

I've added a test that validates we can represent any swagger enum member in the enum in this project.